uint16 device_id
int32 stamp_us
float32 range
uint16 target_id
